content/updates/2020-09/1-Display-labels-in-your-product-exports.md

```diff
@@ -0,0 +1,24 @@
+---
+pim_announcement_img: ./img/Product-export-profile.png
+pim_announcement_alt_img: "Display labels instead of codes in your product exports"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Display labels instead of codes in your product exports
+::: meta-data type="New" features="Collaboration, Productivity" available="mid-September" link-to-doc="../articles/exports.html#update-an-export-profile"
+:::
+
+To save time and improve the reading of exported files, we added an **option in our XLSX and CSV export functionalities to display attribute labels instead of codes in the export files**. Under your product export profiles and also in the quick export, there is now a new button called `With labels`. If enabled, this button will allow you to export product data labels (e.g. attribute labels and values, etc.). This will apply to the **column's headers and file's content**, so instead of displaying attribute and property codes in your files, you can choose to display **attribute values** to make your file more readable if you need to share it with one of your colleague. For the export files, you can even choose to display the content in a specific locale.  
```

Comment:
@felix-couprie I've edited the last part of this paragraph, you mentioned that users can choose a specific locale for the attribute values but this only applies to the export builder right?

Reply:
Users can choose a specific locales amongst those that are activated into their PIM:
- For the QuickExport, locale is based on the one selected in the product grid settings.
- For the ExportBuilder, locale is to be selected in a dedicated field that we added.